---
title: "MICB425 Portfolio"
author: "Ryan Kaien Nah (28632115)"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output: 
  html_document: 
    mathjax: default
    toc: yes
---

```{r setup, include=FALSE, echo=FALSE}
knitr::opts_chunk$set(echo = TRUE)
knitr::read_chunk("DS_assignment20180208.R")
knitr::read_chunk("DS_Friday/26Jan18/26Jan18_SaanichOTU.R")
```

# Module 01

## Module 01 Portfolio Content

* Evidence worksheet_01
    + Completion status: X
    + Comments: 
* Evidence worksheet_02
    + Completion status: X
    + Comments: Fix header formatting.
* Evidence worksheet_03
    + Completion status:
    + Comments: Missing Evidence worksheet_03
* Problem Set_01
    + Completion status: X
    + Comments:
* Problem Set_02
    + Completion status:
    + Comments: Incomplete. Fix header formatting.
* Writing assessment_01
    + Completion status: X
    + Comments:
* Additional Readings
    + Completion status:
    + Comments: Need links.  

### Data Science

* Installation check
    + Completion status: X
    + Comments:
* Portfolio repo setup
    + Completion status: X
    + Comments:
* RMarkdown Pretty PDF Challenge
    + Completion status: X
    + Comments:
* ggplot
    + Completion status: X
    + Comments:

## Data Science Friday 

### Installation check
Use this space to include your installation screenshots.

![](DS_Friday/Assignment1/Assignment1_Figure1.JPG)

![](DS_Friday/Assignment1/Assignment1_Figure2.JPG)

![](DS_Friday/Assignment1/Assignment1_Figure3.JPG)

### Portfolio repo setup
Detail the code you used to create, initialize, and push your portfolio repo to GitHub. This will be helpful as you will need to repeat many of these steps to update your portfolio throughout the course.

```{r gitsetup, eval=FALSE}
$ mkdir MICB425_portfolio #make portfolio directory within desired directory
$ cd MICB425_portfolio #go to new directory
$ git init #designate it as a repo
$ touch ID.txt #create blank ID.txt file
$ git add . #stage all files in new repo for commit
$ git commit -m "First commit" #commit files
$ git remote add origin https://github.com/ryankn/MICB425_portfolio #designate remote repo URL
$ git remove -v #verify remote repo URL
$ git push -u origin master #push local repo to remote repo
```

### RMarkdown pretty PDF challenge
The following is from the activity of recreating the example PDF, with the header levels changed such that they won't appear in the table of contents.

```{r prettypdf, child='DS_Assignment3.Rmd'}
```

### Saanich Metadata Exercises
R code from work for Data Science Friday on 26 Jan 18.

```{r metadata26jan18}
```

### Data Science Friday Assignment Feb 16
R code for Data Science Friday assignment due Friday 16 Feb 18.

```{r DS_Assignment20180208} 
```

## Origins and Earth Systems
### Evidence worksheet 01 
The first thing for any assignment should link(s) to any relevant literature (which should be included as full citations in a module references section below).

[Whitman *et al.* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)

#### Learning objectives
Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems. 

#### General questions
* What were the main questions being asked?  
What is the total number of prokaryotes and the total amount of their cellular carbon, nitrogen, and phosphorus on Earth? How are prokaryotes divided up among various large habitats on Earth? What is the turnover time of prokaryotes in these habitats? What effect does this have on prokaryotic genetic diversity? 

* What were the primary methodological approaches used?  
To make calculation of such figures more plausible, the number of prokaryotes in three large habitats in which current knowledge suggests most prokaryotes reside in were examined, namely: aquatic environments, soil, and the subsurface. All numbers were used from previously published papers reporting various figures like CFU/mL counts, volume estimations, or C content. 

* Summarize the main results or findings.  

Environment            | No. of prokaryotic cells, x 10^28^ | Pg of C in prokaryotes
---------------------- | ---------------------------------- | ----------------------
Aquatic habitats       | 12      | 2.2
Oceanic subsurface     | 355     | 303
Soil                   | 26      | 26
Terrestrial subsurface | 25-250  | 22-215
                Total  | 415-640 | 353-546

  The amount of prokaryotic C is roughly 60-100% of the amount in plants, and that of prokaryotic N and P is likely an entire order of magnitude or two larger than that in plants, around 85–130 Pg and 9–14 Pg for N and P, respectively. 

Habitat                           | No. of prokaryotic cells | Turnover time, days | Cells/yr x 10^29^
--------------------------------- | ------------------------ | --------------------| -------------------
Marine heterotrophs - above 200 m | 3.6 x 10^28^             | 16                  | 8.2
Marine heterotrophs - below 200 m | 8.2 x 10^28^             | 300                 | 1.1
Marine autotrophs                 | 2.9 x 10^27^             | 1.5                 | 7.1
Soil                              | 2.6 x 10^29^             | 900                 | 1.0
Subsurface                        | 4.9 x 10^30^             | 5.5 x 10^5^         | 0.03
Domestic mammals                  | 4.3 x 10^24^             | 1                   | 0.02

  As a result of the large number of cells being produced on a regular basis and combined with their per-gene mutation rate of 4 x 10^-7^, prokaryotes are capable of incredible genetic diversity. 

* Do new questions arise from the results?  
How is the turnover time for the subsurface community so long? At that kind of turnover rate, can that still be constituted as life? Are definitions of prokaryotic species compatible with the knowledge of the rapid mutation/evolution rate of prokaryotes?

* Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  
A myriad of assumptions had to be made, not all of which completely made sense. As one example in relation to the subsurface, the number derived for the number of prokaryotic cells was derived by extrapolating from data collected from the shallower depths of the subsurface, and thus may not be entirely accurate. Additionally, as someone who does not regularly work with numbers of such magnitude, discerning the order of magnitude of fg or Pg masses was challenging. 

### Problem set 01
#### Learning objectives: 
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific questions:
* What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text.  

Habitat    | Abundance
---------- | --------------
Aquatic    | 1.161 x 10^29^
Soil       | 2.556 x 19^29^
Subsurface | 3.8 x 10^30^


* What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including *Prochlorococcus*? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth?  
3.6x10^28^ cells, at 5x10^5^ cells/mL of Cyanobacteria is 8%

  4x10^4^ cells/mL divided by 5x10^5^ cells/mL = 8%

* What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?  
autotrophs fix inorganic carbon e.g. CO~2~ into biomass, heterotrophs assimilate organic carbon,
lithotrophs consume inorganic substrates

* Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth?  
subsurface deep habitats, both terrestrial and marine
terrestrial and marine: up to 4 km, limiting factor is temperature of 125 degrees C
temperature changes about 22 C per km

  Mariana trench - how deep is it? 10.9 km

* Based on information provided in the text and your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height?  

  mount Everest - 8.8 km  
  is anything really alive up in the atmosphere at 77 km? that doesn't seem likely - lack of nutrients or moisture, then there's lots of UV radiation too, sketchy. Let's say 20 km. 

* Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth's biosphere measured in km?  
Thus the vertical distance is about 24 km from top to bottom (tip of mount Everest to 4-5 km under Mariana trench)

* How was annual cellular production of prokaryotes described in Table 7 column four determined? (Provide an example of the calculation)  
Annual cellular production of prokaryotes was calculated based on literature values for population size and population turnover time in days. In the following example calculation, population size is P, turnover time is T, and annual cellular production is A.  

$$A=P*\frac{365}{T}$$
  3.6x10^28^ cells * 365 days / 16 turnovers = 8.2x10^29^ cells/year

* What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?  
assuming carbon efficiency is 20%, so net productivity required is 4x (why not 5? confusion)
assume C content per cell of say 10 fg/cell, which is around 20x10^-30^ Pg/cell
multiply # of cells (3.6x10^28^ cells) by C content per cell, which comes up to about 0.72 Pg C in marine heterotrophs

  0.72*4 = 2.88 Pg C per year

  51 Pg C per year of productivity * 85% = 43 Pg C per year goes to upper 200m
  43/2.88 = 14.9 turnovers a year
  365/14.9 = 24.5 days per turnover

  why does this vary with depth? different production and consumption of C in different habitats

  Carbon assimilation efficiency and carbon content determine turnover rates in the upper 200m of the ocean. The amount of net primary productivity required to sustain prokaryotic turnover is dependent on both C assimilation efficiency and total carbon content of the population, which then sets an upper limit on turnover rates. These vary between habitats because different assimilation efficiencies and total carbon content, as well as the amount of total net primary productivity each habitat zone consumes. 

  also viruses - the viruses kill bugs causing turnover, and carry assessory metabolic genes that when they infect cells, supplement the various metabolic capacities of the community

* How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)  

  4x10^-7^ mutations/generation

  (4x10^-7^)^4^ = 2.56 x 10^-26^ mutations/generation
  
  365/16 = 22.8 turnovers per year

  3.1 x 10^28^ cells * 22.8 = 8.2x10^29^ cells/year

  8.2x10^29^ cells/year x 2.56 x 10^-26^ mutations/generation = 2.1x10^4^ mutations/year

  convert to hours - divide by 365x24 

  2.1x10^4^ / 365 / 24 = 2.4 mutations/hour

  1/2.4 = 0.4 hours/mutation

* Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?  
As a result of their large population size and high mutation rate, prokaryotes are able to very rapidly adapt to a niche, allowing for great genetic diversity. In addition to this, point mutations aren't the only way microbial genomes can adapt - horizontal gene transfer as a result of a variety of causes is a major driving force behind prokaryotic genome diversification as well.   


* What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text? 
The sheer abundance of prokaryotic life lends itself to great genetic diversity,  and this diversity too leads to wide-ranging metabolic capabilities.   

```{r evidenceworksheet_02, child='Module_01/Evidence_worksheet_02.Rmd'}
```

```{r problemset_02, child='Module_01/Problem_set_02.Rmd'}
```

```{r evidenceworksheet_03, child='Module_01/Evidence_worksheet_03.Rmd'}
```

```{r writingassignment_01, child='Module_01/Writing_assignment_01.Rmd'}
```

## Module 01 references
Utilize this space to include a bibliography of any literature you want associated with this module. We recommend keeping this as the final header under each module.

**Whitman WB, Coleman DC, Wiebe WJ**. 1998. Prokaryotes: The unseen majority. *Proc Natl Acad Sci U S A* 95:6578-6583. [PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)

**Nisbet EG, Sleep NH.** 2003. The habitat and nature of early life. *Nature* 409:1083-1091. doi: [10.1038/35059210](http://dx.doi.org/10.1038/35059210).  

**Canfield DE, Glazer AE, Falkowski PG.** 2010. The Evolution and Future of Earth's Nitrogen Cycle. *Science* 330:192-196. doi: [10.1126/science.1186120](https://doi.org/10.1126/science.1186120).  

**Rockstr?m J, Steffen W, Noone K, Persson ?, Chapin FS, Lambin EF, Lenton TM, Scheffer M, Folke C, Schellnhuber HJ, Nykvist B, de Wit CA, Hughes T, van der Leeuw S, Rodhe H, S?rlin S, Snyder PK, Costanza R, Svedin U, Falkenmark M, Karlberg L, Corell RW, Fabry VJ, Hansen J, Walker B, Liverman D, Richardson K, Crutzen P, Foley JA.** 2009. A safe operating space for humanity. *Nature* 461:472-475. doi: [10.1038/461472a](http://dx.doi.org/10.1038/461472a).  

**Schrag DP.** 2012. Geobiology of the Anthropocene, p 425-436. *In* Knoll AH, Canfield DE, Konhauser KO (ed), Fundamentals of Geobiology, 1st ed. Blackwell Publishing Ltd, Oxford, United Kingdom.  

**Falkowski PG, Fenchel T, Delong EF.** 2008. The Microbial Engines That Drive Earth's Biogeochemical Cycles. *Science* 320:1034-1039. doi: [10.1126/science.1153213](http://dx.doi.org/10.1126/science.1153213). 

**Kasting JF, Siefert JL.** 2002. Life and the Evolution of Earth's Atmosphere. *Science* 296:1066-1068. doi: [10.1126/science.1071184](https://doi.org/10.1126/science.1071184).  

# Module 02

## Module 02 Portfolio Content

* Evidence worksheet_04
    + Completion status:
    + Comments: Missing Evidence worksheet_04
* Problem Set_03
    + Completion status: X
    + Comments:
* Writing assessment_02
    + CANCELED
* Additional Readings
    + Completion status:
    + Comments 

## Remapping the Body of the World

```{r evidenceworksheet_04, child='Module_02/Evidence_worksheet_04.Rmd'}
```

```{r problemset_03, child='Module_02/Problem_set_03.Rmd'}
```

## Module 02 References  
**Wooley JC, Godzik A, Friedberg I.** 2010. A primer on metagenomics. PLoS Comp Biol **6**:e1000667.  
**Martinez A, Bradley AS, Waldbauer JR, Summons RE, DeLong EF.** 2007. Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host. Proc Natl Acad Sci U S A **104**:5590-5595.  
**Madsen EL.** 2005. Identifying microorganisms responsible for ecologically significant biogeochemical processes. Nat Rev **3**:439-446.  

# Module 03

## Module 03 Portfolio Content

* Evidence worksheet_05
    + Completion status: X
    + Comments: Provide header for Evidence worksheet_05
* Problem set_04
    + Completion status: X
    + Comments:
* Writing Assessment_03
    + Completion status:
    + Comments:
* Additional Readings
    + Completion status:
    + Comments 

### Project 1

* CATME account setup and survey
    + Completion status: X
    + Comments: 
* CATME interim group assessment
    + Completion status: X
    + Comments:
* Project 1
    + Report (80%): 
    + Participation (20%):

## Microbial Species Concepts

```{r problemset_04, child='Module_03/Problem_set_04.Rmd'}
```

```{r evidenceworksheet_05, child='Module_03/Evidence_worksheet_05.Rmd'}
```

```{r writingassignment_03, child='Module_03/Writing_assignment_03.Rmd'}
```

## Project 1
See .Rmd and knitted .html in Module_03 directory.

## Module 03 References
Didn't have time, sorry!

# Module 04

## Module 04 Portfolio Content
### Project 2

* CATME final group assessment
    + Completion status: 
    + Comments:
* Project 2
    + Report (80%): 
    + Participation (20%):
    
## Project 2
See .Rmd and knitted .html in Module_04/Markers directory.