---
title: "Project_02"
author: "Group 4"
date: "April 25, 2018"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

#Project 2
##**Abstract**
Sequences for nitrite reductase (*nirS*) gene were obtained from various depths of Saanich Inlet, a model ecosystem for the effects of growing oxygen minimum zones in the open ocean. Tree-based Sensitive and Accurate Protein Profiler (TreeSAPP) was implemented for automated reconstruction of the nitrogen cycle along defined redox gradients in Saanich Inlet for metagenomic and metatranscriptomic assemblies of *nirS*. Increased DNA and RNA abundance of *nirS* was found with increasing depth under the suboxic and anoxic conditions, and no expression was observed at oxic conditions. *Gammaproteobacteria*, *Planctomycetia*, *Deltaproteobacteria* and *Alphaproteobacteria* groups predominante the overall *nirS* expression in the Saanich Inlet. It was found that the distribution of *nirS* throughout the water column may be due to *Planctomycetia* expressing anammox-type *nirS* that thrives in lower suboxic zones, and *Gammaproteobacteria* expressing denitrification-type *nirS* which is highly suitable for denitrification in anoxic zones and upper suboxic zones. In addition, *nirS* OTUs have been shown to be station and time-dependent. As the Saanich Inlet is a seasonally anoxic fjord, the DNA and RNA abundances for denitrification genes should fluctuate with time as the water column moves in and out of anoxia. In this project, we investigate the *nirS* gene in the denitrification pathway to assess in terms of taxonomic rank, abundance and expression along the redoxcline in Saanich Inlet and propose future directions to expand on our results.


##**Introduction**
Saanich Inlet acts as an excellent model system for oxygen minimum zones (OMZs), due to its seasonally anoxic character. A shallow sill separates the body of the inlet from the Strait of Georgia, and traps deep water throughout the spring and summer. Microbial activity quickly depletes dissolved oxygen below 100m, when large amounts of organic matter sink during the spring phytoplankton bloom. Then, during turbulent winter weather, the mixed layer depth increases and the anoxic deep water is exchanged for new, oxygen rich water over the shallow sill [1]. These processes repeat annually, resulting in a cycle of deep water replenishment, followed by the development of an anoxic region. The predictable nature of changing oxygen concentrations, combined with excellent sampling conditions make Saanich Inlet a first-rate model OMZ ecosystem. 


The expansion of OMZs is expected to cause drastic shifts in global marine microbial community compositions, and therefore their functional capabilities. There is a demand for models of how OMZs form due to a global increase in OMZ size and intensity [2]. OMZs are bands of hypoxic water between 100m and 1000m in depth. Microbial respiration fueled by sinking organic matter from the euphotic zone depletes dissolved oxygen all the way to anoxia in extreme circumstances. The strength and depth range of these zones has been shown to be increasing on average due to anthropogenic climate change. Oxygen is not only a vital nutrient for macroscopic life, but also a determining factor in how microbial communities function. When oxygen is not available, bacteria can reduce other chemicals for energy production. In the absence of oxygen, nitrate is often the first choice. 

Nitrogen is cycled biologically on an immense scale globally. It is a necessary component of all life in relatively large amounts. For practical reasons, nitrogen is often divided into two main pools: fixed nitrogen, which is soluble in water and bioavailable, and unfixed nitrogen, which is in a gaseous form and diffuses to the atmosphere. Cycling between these two pools occurs by three main processes. Nitrogen fixation is the conversion of N~2~ gas to NH~4~^+^ by diazotrophs. In the ocean, diazotrophs are mostly cyanobacteria, which require high light and iron availability. This limits nitrogen fixation to relatively small portions of the surface ocean. The two processes that counter nitrogen fixation, releasing fixed nitrogen back to the atmosphere, are annamox and denitrification. Annamox converts NO3- and NH~4~^+^ to N~2~ but is thought to contribute less to bioavailable nitrogen loss than denitrification. Denitrification is the stepwise reduction of NO~3~^-^ to N~2~. Both annamox and denitrification require extremely low O~2~ environments and are prevalent in oceanic OMZs.


The ability of a community to denitrify is dependant on the genetic presence of a full nitrogen reduction pathway. Often, microbes will not contain the genes or transcribe the genes necessary for all intermediate steps of a metabolic pathway, instead opting to rely on other organisms for the processing of the steps they lack [3]. *NirS* is of particular interest because it catalyzes the conversion of nitrite (NO~2~^-^) to nitric oxide (NO) (Fig 1), the point of no return in the denitrification pathway where a bioavailable ion is converted to a gas. Here the distribution of *nirS* genes and transcripts across depths and across taxa is analysed in a representative summer Saanich Inlet water column.


![**Figure 1.** Nitrogen Cycle](N_cycle.png)


##**Methods**
Environmental samples were collected through time-series monitoring in Saanich Inlet on a monthly-basis aboard the MSV John Strickland at station S3 (48°35.500 N, 123°30.300 W) [4, 5]. Samples for large volume (LV) SSU rRNA gene tags, metagenomics, metatranscriptomics, and metaproteomics were taken from six depths spanning the oxycline (10, 100, 120, 135, 150, 165, and 200 m). Samples for high-resolution (HR) SSU rRNA gene tag sequencing were taken at 16 depths along the oxycline (10, 20, 40, 60, 75, 80, 90, 97, 100, 110, 120, 135, 150, 165, 185 and 200 meters). Water was collected with Go-Flo bottles and dissolved gas samples were taken immediately after the bottles were taken onboard the ship. Next, the remaining water was quickly drained into 2L Nalgene bottles through sterilized silicon tubing to minimise changes in microbial gene expression.A 0.22 μm Sterivex filter was used to collect biomass from water samples. Genomic DNA and RNA were then extracted from cells on these filters. RNA was reverse transcribed into cDNA and both genomic DNA and cDNA were used in the construction of shotgun Illumina libraries. Sequencing data were generated on the Illumina HiSeq platform with 2x150bp technology.


The resulting reads were processed and quality controlled at the Joint Genome Institute using the IMG/M pipeline. Metagenomes were assembled and processed using MetaPathways 2.5 at UBC.


Tree-based Sensitive and Accurate Protein Profiler (TreeSAPP) was implemented for automated reconstruction of the nitrogen cycle along defined redox gradients in Saanich Inlet using Google Cloud services for metagemomic (MetG, DNA) and metatranscriptomic (MetaT, RNA) assemblies of *nirS*. 


Reads per kilo base per million mapped reads (RPKM) was used to quantify gene expression by normalizing for total read length and the number of sequencing reads when comparing the gene coverage values. This corrects the difference in sampling sequencing depth and gene length [6]. The formula below indicates numReads as the number of reads mapped to a gene sequence, geneLength as the length of the gene sequence and totalNumReads as the total number of mapped reads for a sample: 


$$RPKM =  \frac{numReads}{\frac{geneLength}{10^3}\times \frac{totalNumReads}{10^6}}$$


The following commands were used in the pipeline:
```{r eval =FALSE}
#!/bin/bash
cd /home/enenn/
export WISECONFIGDIR=/home/connor/TreeSAPP//data//genewise_support_files//wisecfg
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_10m_DNA.scaffolds.fasta  --rpkm \
-r bucket/MetaG_reads/SI072_LV_10m.anqdp.fastq.gz -o treesapp/treesapp_out_G_10m
rm treesapp/treesapp_out_G_10m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_G_10m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_10m_DNA.scaffolds.fasta --rpkm \
-r bucket/MetaT_reads/SI072_LV_10m.qtrim.3ptrim.artifact.rRNA.clean.fastq.gz \
-o treesapp/treesapp_out_T_10m
rm treesapp/treesapp_out_T_10m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_T_10m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_100m_DNA.scaffolds.fasta  --rpkm \
-r bucket/MetaG_reads/SI072_LV_100m.anqdp.fastq.gz -o treesapp/treesapp_out_G_100m
rm treesapp/treesapp_out_G_100m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_G_100m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_100m_DNA.scaffolds.fasta --rpkm \
-r bucket/MetaT_reads/SI072_LV_100m.qtrim.3ptrim.artifact.rRNA.clean.fastq.gz \
-o treesapp/treesapp_out_T_100m
rm treesapp/treesapp_out_T_100m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_T_100m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_120m_DNA.scaffolds.fasta  --rpkm \
-r bucket/MetaG_reads/SI072_LV_120m.anqdp.fastq.gz -o treesapp/treesapp_out_G_120m
rm treesapp/treesapp_out_G_120m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_G_120m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_120m_DNA.scaffolds.fasta --rpkm \
-r bucket/MetaT_reads/SI072_LV_120m.qtrim.3ptrim.artifact.rRNA.clean.fastq.gz \
-o treesapp/treesapp_out_T_120m
rm treesapp/treesapp_out_T_120m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_T_120m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_135m_DNA.scaffolds.fasta  --rpkm \
-r bucket/MetaG_reads/SI072_LV_135m.anqdp.fastq.gz -o treesapp/treesapp_out_G_135m
rm treesapp/treesapp_out_G_135m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_G_135m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_135m_DNA.scaffolds.fasta --rpkm \
-r bucket/MetaT_reads/SI072_LV_135m.qtrim.3ptrim.artifact.rRNA.clean.fastq.gz \
-o treesapp/treesapp_out_T_135m
rm treesapp/treesapp_out_T_135m/RPKM_outputs/*.sam &

chmod -Rf treesapp/treesapp_out_T_135m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_150m_DNA.scaffolds.fasta  --rpkm \
-r bucket/MetaG_reads/SI072_LV_150m.anqdp.fastq.gz -o treesapp/treesapp_out_G_150m
rm treesapp/treesapp_out_G_150m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_G_150m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_150m_DNA.scaffolds.fasta --rpkm \
-r bucket/MetaT_reads/SI072_LV_150m.qtrim.3ptrim.artifact.rRNA.clean.fastq.gz \
-o treesapp/treesapp_out_T_150m
rm treesapp/treesapp_out_T_150m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_T_150m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_165m_DNA.scaffolds.fasta  --rpkm \
-r bucket/MetaG_reads/SI072_LV_165m.anqdp.fastq.gz -o treesapp/treesapp_out_G_165m
rm treesapp/treesapp_out_G_165m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_G_165m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 -p pe \
-i bucket/MetaG_assemblies/SI072_LV_165m_DNA.scaffolds.fasta --rpkm \
-r bucket/MetaT_reads/SI072_LV_165m.qtrim.3ptrim.artifact.rRNA.clean.fastq.gz \
-o treesapp/treesapp_out_T_165m
rm treesapp/treesapp_out_T_165m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_T_165m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 \
-i bucket/MetaG_assemblies/SI072_LV_200m_DNA.scaffolds.fasta  --rpkm \
-r bucket/MetaG_reads/SI072_LV_200m.anqdp.fastq.gz -o treesapp/treesapp_out_G_200m
rm treesapp/treesapp_out_G_200m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_G_200m 777 &
time treesapp.py -T 8 --verbose --delete -t D0302 \
-i bucket/MetaG_assemblies/SI072_LV_200m_DNA.scaffolds.fasta --rpkm \
-r bucket/MetaT_reads/SI072_LV_200m.qtrim.3ptrim.artifact.rRNA.clean.fastq.gz \
-o treesapp/treesapp_out_T_200m
rm treesapp/treesapp_out_T_200m/RPKM_outputs/*.sam &
chmod -Rf treesapp/treesapp_out_T_200m 777
```


Interactive Tree of Life (iTOL) version 4.2 was used for the display, annotation and management of the phylogenetic trees produced for the *nirS* gene within the denitrification pathway [7].


DNA and RNA abundances of nirS were compared across depth and different taxa. Analysis was completed in R v3.4.3 [8] using the following packages and the TreeSAPP output was loaded into R and combined into a single dataframe, nirS.all using the following commands:

```{r}
library(tidyverse)
library(cowplot)

#DNA data
nirS.DNA.10m = read_tsv("nirS_DNA_10m_marker_contig_map.tsv") %>%
  select(Tax.DNA.10 = Confident_Taxonomy, Abund.DNA.10 = Abundance, Query)
nirS.DNA.100m = read_tsv("nirS_DNA_100m_marker_contig_map.tsv") %>%
  select(Tax.DNA.100 = Confident_Taxonomy, Abund.DNA.100 = Abundance, Query)
nirS.DNA.120m = read_tsv("nirS_DNA_120m_marker_contig_map.tsv") %>%
  select(Tax.DNA.120 = Confident_Taxonomy, Abund.DNA.120 = Abundance, Query)
nirS.DNA.135m = read_tsv("nirS_DNA_135m_marker_contig_map.tsv") %>%
  select(Tax.DNA.135 = Confident_Taxonomy, Abund.DNA.135 = Abundance, Query)
nirS.DNA.150m = read_tsv("nirS_DNA_150m_marker_contig_map.tsv") %>%
  select(Tax.DNA.150 = Confident_Taxonomy, Abund.DNA.150 = Abundance, Query)
nirS.DNA.165m = read_tsv("nirS_DNA_165m_marker_contig_map.tsv") %>%
  select(Tax.DNA.165 = Confident_Taxonomy, Abund.DNA.165 = Abundance, Query)
nirS.DNA.200m = read_tsv("nirS_DNA_200m_marker_contig_map.tsv") %>%
  select(Tax.DNA.200 = Confident_Taxonomy, Abund.DNA.200 = Abundance, Query)

#RNA data
nirS.RNA.10m = read_tsv("nirS_RNA_10m_marker_contig_map.tsv") %>%
  select(Tax.RNA.10 = Confident_Taxonomy, Abund.RNA.10 = Abundance, Query)
nirS.RNA.100m = read_tsv("nirS_RNA_100m_marker_contig_map.tsv") %>%
  select(Tax.RNA.100 = Confident_Taxonomy, Abund.RNA.100 = Abundance, Query)
nirS.RNA.120m = read_tsv("nirS_RNA_120m_marker_contig_map.tsv") %>%
  select(Tax.RNA.120 = Confident_Taxonomy, Abund.RNA.120 = Abundance, Query)
nirS.RNA.135m = read_tsv("nirS_RNA_135m_marker_contig_map.tsv") %>%
  select(Tax.RNA.135 = Confident_Taxonomy, Abund.RNA.135 = Abundance, Query)
nirS.RNA.150m = read_tsv("nirS_RNA_150m_marker_contig_map.tsv") %>%
  select(Tax.RNA.150 = Confident_Taxonomy, Abund.RNA.150 = Abundance, Query)
nirS.RNA.165m = read_tsv("nirS_RNA_165m_marker_contig_map.tsv") %>%
  select(Tax.RNA.165 = Confident_Taxonomy, Abund.RNA.165 = Abundance, Query)
nirS.RNA.200m = read_tsv("nirS_RNA_200m_marker_contig_map.tsv") %>%
  select(Tax.RNA.200 = Confident_Taxonomy, Abund.RNA.200 = Abundance, Query)

#Combine all data frames
nirS.all = nirS.DNA.100m %>%
  full_join(nirS.DNA.120m, by = "Query") %>%
  full_join(nirS.DNA.135m, by = "Query") %>%  
  full_join(nirS.DNA.150m, by = "Query") %>%  
  full_join(nirS.DNA.165m, by = "Query") %>%
  full_join(nirS.DNA.200m, by = "Query") %>%
  full_join(nirS.RNA.100m, by = "Query") %>%
  full_join(nirS.RNA.120m, by = "Query") %>%
  full_join(nirS.RNA.135m, by = "Query") %>%
  full_join(nirS.RNA.150m, by = "Query") %>%
  full_join(nirS.RNA.165m, by = "Query") %>%
  full_join(nirS.RNA.200m, by = "Query") %>%
  mutate(Taxonomy = ifelse(!is.na(Tax.DNA.100), Tax.DNA.100,
                    ifelse(!is.na(Tax.DNA.120), Tax.DNA.120,
                    ifelse(!is.na(Tax.DNA.135), Tax.DNA.135,
                    ifelse(!is.na(Tax.DNA.150), Tax.DNA.150,
                    ifelse(!is.na(Tax.DNA.165), Tax.DNA.165,
                    ifelse(!is.na(Tax.DNA.200), Tax.DNA.200,
                    ifelse(!is.na(Tax.RNA.100), Tax.RNA.100,
                    ifelse(!is.na(Tax.RNA.120), Tax.RNA.120,                           
                    ifelse(!is.na(Tax.RNA.135), Tax.RNA.135,
                    ifelse(!is.na(Tax.RNA.150), Tax.RNA.150,
                    ifelse(!is.na(Tax.RNA.165), Tax.RNA.165,
                    ifelse(!is.na(Tax.RNA.200), Tax.RNA.200,
                           "unclassified"))))))))))))) %>%
  select(-starts_with("Tax.")) %>%
  gather("Key", "Abundance", starts_with("Abund")) %>%
  separate(Key, c("Key","Type","Depth_m"), by = ".") %>%
  select(Depth_m, Type, Abundance, Taxonomy, Query) %>%
  mutate(Depth_m = as.numeric(Depth_m)) %>%
  separate(Taxonomy, into = c("Domain", "Phylum", "Class", "Order", 
                              "Family","Genus", "Species"), sep=";") 
```

##**Results**


```{r fig2, fig.height = 8, fig.width = 6, out.width = "50%", echo = FALSE}
#Combine and sum all abundances at a certain depth
nirS.abund = aggregate(Abundance ~ Type + Depth_m, data = nirS.all, sum)

#Plot DNA vs. RNA Abundance
ggplot(nirS.abund, aes(x = Abundance, y = Depth_m)) +
  geom_path(aes(colour = Type), size = 1.5) +
  scale_y_reverse(lim=c(200,0)) +
  ggtitle("Abundance of nirS gene (DNA vs. RNA) at different depths") + 
  theme(plot.title = element_text(face = "bold", hjust=0.5)) +
  xlab("Relative Abundance") +
  ylab("Depth (m)")
```


**Figure 2.** DNA vs. RNA relative abundance of nirS gene and transcripts from 10 to 200 metres depth in Saanich Inlet. 


As shown in Figure 2, the DNA and RNA abundances of the *nirS* gene vary significantly by depth, with the general trend being that abundance increases with depth below 100m. Above 100m, the only depth that was measured was 10m, and there was neither DNA nor RNA found for the *nirS* gene. The abundance of DNA is higher than that of RNA, down to 150m, where RNA abundance increases significantly by about one order of magnitude from 82 to 750. This means that transcription of nitrite reductase, *nirS*, increases under anoxic conditions, which start around 150m. RNA abundance reaches a maximum at 165m, but the DNA abundance decreases by a sizeable amount at this depth, which could mean that there is lower gene diversity, but high rates of transcription in each organism, leading to more RNA. DNA abundance reaches a maximum at 200m, and RNA abundance decreases slightly, which means that there could be more diversity, but lower transcription rates for each organism. Overall, RNA abundances are significantly higher than DNA abundance below 150m, but above this depth, DNA abundances are higher, meaning that the nirS gene is present in cells, but it is not transcribed at high rates. 


```{r fig3, fig.height = 10, fig.width = 18, echo = FALSE}
#Load phyloseq data
load("mothur_phyloseq.RData")
metadata = data.frame(mothur@sam_data)
#Select NO3, NO2, O2 + set line colours
mn <- metadata %>% 
  gather(Nutrient, Concentration, NO3_uM, NO2_uM, O2_uM) %>% 
  select(Depth_m, Nutrient, Concentration)
colours = c("NO3_uM" = "green", "NO2_uM" = "red", "O2_uM" = "blue") 

#Plot by Order
plot1 = nirS.all %>% 
  mutate(Order = ifelse(is.na(Order), "unclassified", Order)) %>% 
  ggplot(aes(x = Order, y = Depth_m)) +
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance), colour = Type), 
             position = position_dodge(0.5)) +
  theme(axis.text.x = element_text(size = 12, hjust=0.5))+
  theme(axis.text.x = element_text(angle=-90,hjust=1,vjust=0.5))+
  scale_x_discrete(labels = function(x) str_wrap(x, width = 6))+
  scale_y_reverse(lim=c(200,10)) +
  scale_size_continuous(name = "Abundance")+
  labs(y = "Depth (m)",
       x = "Order")

#Plot phyloseq metadata
plot2 = ggplot()+
  geom_point(data = mn, aes(x = log10(Concentration), y = Depth_m, colour = Nutrient))+
  geom_path(data = mn,aes(x = log10(Concentration), y = Depth_m, colour = Nutrient)) +
  scale_color_manual(values=colours, 
                     labels=c(expression(paste("NO"[2])),
                              expression(paste("NO"[3])), 
                              expression(paste("O"[2]))))+
  scale_y_reverse(lim=c(200,10))+
  labs(y = "Depth (m)",
       x = expression(paste("Log"[10], " Concentration", " (μM)")))

plot_grid(plot2, plot1, labels=c("A","B"), rel_widths = c(1/4, 3/4), align = 'h')
```


**Figure 3.** A: Change in nitrite, nitrate, and oxygen concentrations (Log~10~ scale) from 10 to 200 metres depth in Saanich Inlet. B: Change in DNA vs. RNA abundance of the nirS gene from 10 to 200 metres depth across different Orders in Saanich Inlet. Note that the Log~10~ was used for concentration as the nitrite concentration was relatively low compared to nitrate and oxygen. 


```{r fig4, fig.height = 10, fig.width = 18, echo = FALSE}
#Plot by Class
plot3 = nirS.all %>% 
  mutate(Class = ifelse(is.na(Class), "unclassified", Class)) %>% 
  ggplot(aes(x = Class, y = Depth_m)) +
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance), colour = Type), 
             position = position_dodge(0.5)) +
  theme(axis.text.x = element_text(size = 12, hjust=0.5))+
  theme(axis.text.x = element_text(angle=-90,hjust=1,vjust=0.5))+
  scale_x_discrete(labels = function(x) str_wrap(x, width = 6))+
  scale_y_reverse(lim=c(200,10)) +
  scale_size_continuous(name = "Abundance")+
  labs(y = "Depth (m)",
       x = "Class")

plot_grid(plot2, plot3, labels=c("A","B"), rel_widths = c(1/4, 3/4), align = 'h')
```


**Figure 4.** A: Change in nitrite, nitrate, and oxygen concentrations (Log~10~ scale) from 10 to 200 metres depth in Saanich Inlet. B: Change in DNA vs. RNA abundance of the nirS gene from 10 to 200 metres depth across different Classes in Saanich Inlet. Note that the Log~10~ was used for concentration as the nitrite concentration was relatively low compared to nitrate and oxygen. 


```{r fig5, fig.height = 10, fig.width = 18, echo = FALSE}
#Plot by Family
plot4 = nirS.all %>% 
  mutate(Class = ifelse(is.na(Family), "unclassified", Family)) %>% 
  ggplot(aes(x = Family, y = Depth_m)) +
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance), colour = Type), 
             position = position_dodge(0.5)) +
  theme(axis.text.x = element_text(size = 12, hjust=0.5))+
  theme(axis.text.x = element_text(angle=-90,hjust=1,vjust=0.5))+
  scale_x_discrete(labels = function(x) str_wrap(x, width = 6))+
  scale_y_reverse(lim=c(200,10)) +
  scale_size_continuous(name = "Abundance")+
  labs(y = "Depth (m)",
       x = "Family")

plot_grid(plot2, plot4, labels=c("A","B"), rel_widths = c(1/4, 3/4), align = 'h')
```


**Figure 5.** A: Change in nitrite, nitrate, and oxygen concentrations (Log~10~ scale) from 10 to 200 metres depth in Saanich Inlet. B: Change in DNA vs. RNA abundance of the nirS gene from 10 to 200 metres depth across different Families in Saanich Inlet. Note that the Log~10~ was used for concentration as the nitrite concentration was relatively low compared to nitrate and oxygen. 


```{r fig6, fig.height = 10, fig.width = 18, echo = FALSE}
# Libraries
library(tidyverse)
library(phyloseq)
library(vegan)

# Loading in Data
load("mothur_phyloseq.RData")
load("qiime2_phyloseq.RData")


#Fig 6

mothur_percent = 	mothur %>% transform_sample_counts(function(x) 100 * x/sum(x)) 

mothur_percent %>% 
  plot_bar(fill="Class") + 
  geom_bar(aes(fill=Class), stat="identity") +
  labs(x = "Sample depth", y = "Relative abundance (%)", 
       title = "Class from 10 to 200 m in Saanich Inlet")
```


**Figure 6. Relative abundances of Class at various depths in Saanich Inlet.** The most abundant class with 90%-100% relative abundance at all depth are *Acidlmicroblia*, and *Actinobacteria*, The intermediate abundance of class between 53%-85% relative abundance are mainly unclassified *Archaea*, *Deltaproteobacteria*, *Epsilonproteobacteria*, *Gammaproteobacteria*. The Phyla with the lowest relative abundances with approximately 2%-20% are mainly *Marinimicrobia*, *Planctomycetes*, *Varrucomicrobia* and unclassified *Proteobacteria*.


![**Figure 7. nirS relative DNA and RNA expression in 100m, 120m, 135m, 150m, 165m, 200m depth relative to taxa.** Taxonomic classification of organisms are divided by different colours, shown in a wheel at the end of the tree branch. The green and red bars extending out from the organism names depict relative DNA and RNA abundances of nirS gene, respectively. 10m is not shown as there were no DNA or RNA expression of nirS for all analyzed organisms.](iTOL_outputs.png)


As shown in Figures 4 and 7, diversity of organisms containing the *nirS* gene varies with depth. *Gammaproteobacteria* genera *Colwellia*, *Oleispira*, *Olephilus*, *Thalassomonas*, *Ridgeia* and *Sedimenticola* have consistent relative level of DNA expression compared to other organisms at all analyzed depths. At 100m, the level of RNA transcription follows the trend of DNA expression in members of the *Gammaproteobacteria*, *Planctomycetia* and *Deltaproteobacteria* class. At 120m, the RNA transcripts of *nirS* for *Gammaproteobacteria* are less than DNA. However, the inverse is true for some members of the *Alphaproteobacteria*, *Anaerolineae*, *Planctomycetia*, *Deltaproteobacteria*, *Deinococci*, *Bacteroidetes*, *Aquificae*, *Epsilonproteibacteria*, *Aciditiobacillia* classes, in which the RNA transcription level is much higher than DNA expression. Of note, *Planctomycemia* have the highest RNA abundance. At 135m, the same trend in relative DNA and RNA is observed as at 120m. At 150m, the O~2~ concentration decreases to zero, there is high *nirS* expression in *Gammaproteobacteria*, and medium to low expression in *Alphaproteobacteria*. RNA levels follows the trend of DNA expression. At 165m, NO~3~ concentration also decreases to zero. *NirS* expression decreases in *Alphaproteobacteria*, but slightly increases in *Planctomycetia* and *Deltaproteobacteria*. RNA abundances still follows the trend of DNA. At 200m, the O~2~, NO~3~ and NO~2~ concentrations are zero. RNA is lower than DNA in all previously mentioned *Gammaproteobacteria* species except for *Olephilus*. However, RNA abundance is slightly higher than DNA in the *Anaerolineae* class. We must mention that the systemic *nirS* expression observed in all species at 135m, 150m and 200m is an artifact. There is an organism identified at a higher taxonomic level with *nirS* expression, but the pipeline cannot differentiate it further. Thus, *nirS* expression was conferred to all species under that taxonomic level.


##**Discussion**
Denitrification is a sequential, four-step process, nitrate (NO~3~^-^), a fixed inorganic species, is reduced to dinitrogen gas (N~2~). This process only occurs under suboxic and anoxic conditions (1-2% oxygen saturation), where the most favourable terminal electron acceptor (TEA), oxygen (O~2~) is limiting and organisms take up the next favourable TEA, NO~3~^-^ [9,10]. Our gene of interest, *nirS*; nitrite reductase, is involved in the reduction of nitrite (NO~2~^-^) to nitric oxide (NO) in the second step of denitrification and also contains cytochrome *cd1* [11, 12]. Within the water column of Saanich Inlet, dissolved oxygen concentrations decrease steadily with depth until 150m,  where values quickly approach zero (Figures 3A, 4A, 5A.). This results to a sharp increase in DNA and RNA abundance, consistent with the fact that denitrification happens under suboxic and anoxic conditions (Figures 3A, 4A, 5A.). 


This nitrite reduction process is shared with another nitrite reductase, nirK, which contains copper instead of the cytochrome *cd1* used in *nirS* [11]. Gene analyses from experiments done in other other anoxic/suboxic basins, such as the Baltic Sea, Black Sea and the San Francisco Bay, have shown that gene abundances were about 1-2 orders of magnitude higher for nirS than in *nirK* in the San Francisco Bay [13]. OTU abundances based on a 95% sequence similarity cutoff for all experiments were also higher for *nirS* compared to *nirK* by up to five times at certain stations in the Baltic Sea and the San Francisco Bay [13,14]. However, in the Black Sea, the OTU abundance of *nirS* is either approximately the same, or up to five times lower than the abundance of nirK [15]. Alpha-diversity and richness of *nirS* and *nirK* OTUs are largely dependent on the station and time of year, which is seen in all three experiments [13,14,15]. 
	
	
Our TreeSAPP analysis showed that the *Gammaproteobacteria*, *Planctomycetia*, *Deltaproteobacteria* and *Alphaproteobacteria* groups predominante the overall *nirS* expression in the Saanich Inlet. There are two noteworthy points to highlight from the results. First, we observed that the *Gammaproteobacteria* is present in the Saanich Inlet at relatively high abundances of 50-85% at all depths (Figure 6.), which corresponds to also having the highest relative abundance of *nirS* gene copies at all analyzed depths (Figure 7.). This observation suggests a high level of denitrifying metabolic activity within this group. Studies with pure cultures and seawater enrichments have shown that the *Gammaproteobacteria* may maintain a higher ribosome content per cell [16,17], with which the increased rRNA levels may permit these bacteria to better respond to rapid changes in oxygen concentration in the seasonally anoxic Saanich Inlet [18]. Secondly, despite *Gammaproteobacteria* exhibiting relatively high level of *nirS* DNA, lower RNA abundance was detected at 120m and 135m, where non-*Proteobacteria* groups such as *Planctomycetia* dominate (Figure 7.). It is also worthy to note that *Planctomycetia* composes less than 10% of overall organism abundance at these depths (Figure 6.), but disproportionality accounts for a large portion of *nirS* expression (Figure 7.). *Planctomycetia* is known to mainly carry out anammox metabolism [19]. Gene-sequencing studies investigating expression of anammox sequences have shown that *nirS* genes expressed consistently at lower suboxic zones, such as 120m and 135m in Saanich, are closely allied to the environmental anammox clade [20]. The presence of anammox bacteria in the lower suboxic zone may be due to the flux of ammonium from the sulfide zone that affects these depths [20]. 


Meanwhile, *Gammaproteobacteria* may express *nirS* that is more closely related to denitrification pathways, which studies have shown to be more abundant in anoxic zones, and also in upper suboxic zones (O~2~ > 10 µM) [20]. While anoxic environments are the most suitable for denitrification, studies have also suspected that the enzymatic machinery in *Alpha-* and *Gammaproteobacteria* for anaerobic denitrification is likely to be similar to aerobic denitrification, allowing possible denitrification in upper suboxic zones [21]. In other words, the distribution of *nirS* throughout the water column may be due to *Planctomycetia* expressing anammox-type *nirS* that thrives in lower suboxic zones, and *Gammaproteobacteria* expressing denitrification-type *nirS* which is highly suitable for denitrification in anoxic zones and upper suboxic zones. 


In this investigation, we have seen the abundance and expression of *nirS* vary with taxa and location. This is one of a number of genes involved in the distributed metabolism of the N cycle. It is likely advantageous for the genes in the N cycle to be specialized in order to interact with the many forms nitrogen can adopt. It is also advantageous to distribute these genes across specific taxa in the community since this increases the adaptability of microbial communities in response to their ecosystem. In addition, transitions in the oxidation state of nitrogen occur as a result of thermodynamically driven microbial reactions. It would be energetically irrational for an organism inhabiting anoxic regions of the ocean to actively express genes involved in nitrification. Microbial communities involved in the N cycle have evolved in symbiosis to achieve energetically efficient niches. For example, consider the diazotrophic cyanobacterium *UCYN-A*. *UCYN-A* is lacking photosystem II, RubisCo and the tricarboxylic acid cycle, meaning it is not capable of fixing carbon. Instead, it has formed a symbiotic relationship with a prymnesiophyte where *UCYN-A* receives fixed carbon in exchange for nitrogen [22]. Until the identification of *UCYN-A*, nitrogen fixation in the oceans had been attributed mainly to *Trichodesmium* [23]. Further investigation will likely yield even more discoveries indicating the vastness of the N cycle. Ecological studies of microbial community dynamics could provide further insight into the functioning N cycle.


It should be noted that some genes in the nitrogen cycle have similar functions. For example, the nirK gene also performs nitrite reduction, like nirS. As alpha-diversity and richness of *nirS* and *nirK* OTUs have been shown to be station and time-dependent [13,14,15], this should also apply to Saanich Inlet, as it is a seasonally anoxic fjord, which can affect the distribution of denitrification pathways at different parts of the year as oxygen concentrations in the water column change. As a result, DNA and RNA abundances for denitrification genes should fluctuate with time as the water column moves in and out of anoxia. Accordingly, an avenue for future research would be to compare *nirS* abundance to *nirK* abundance, by obtaining the *nirK* dataset using TreeSAPP. 


Another method of microbial NO~2~^-^ loss in the water column is through anaerobic ammonium oxidation (anammox), which converts either NO~2~^-^ or NH~4~^+^ to dinitrogen gas (N~2~) using either the *Hao* (hydroxilamine reductase) or *Hzo* (hydroxylamine oxidoreductase) gene [12]. Autotrophic bacteria have been shown to undergo anammox [24], meaning that they could potentially have a selective advantage where organic carbon flux from the surface is limited, given that they can produce their own organic carbon. In addition, there are also bacteria that can use an alternative organic carbon source, propionate, for growth [25]. Although bacteria that undergo anammox are slow-growing [26], these alternative source of organic carbon, means that they can potentially have a competitive advantage over nitrite reducers in certain areas of the ocean. Although our analysis is primarily focused on denitrification pathways, comparing *nirS* and *nirK* abundance with *Hao* and *Hzo* abundances would help determine if annamox has an effect on denitrification rates in Saanich Inlet.	


##**References**
1. Jetten MSM, Strous M, van de Pas-Schoonen K, Kramer EHM, Muyzer G, Fuerst JA, Kuenen JG, Webb R, Logemann S. 1999. Missing lithotroph identified as new planctomycete. Nature. 400(6743):446-449. doi:10.1038/22749.
2. Stramma L, Schmidtko S, Levin LA, Johnson GC. 2010. Ocean oxygen minima expansions and their biological impacts. Deep Sea Res Part 1 Oceanogr Res Pap. 57(4):587-595. doi:10.1016/j.dsr.2010.01.005 
3. Giovannoni SJ. 2017. SAR11 bacteria: The most abundant plankton in the oceans. Ann Rev Mar Sci. 9(1):231-255. doi:10.1146/annurev-marine-010814-015934.
4. Torres-Beltrán M, Hawley AK, Capelle D, Zaikova E, Walsh DA, Mueller A, Finke J. 2017. A compendium of geochemical information from the Saanich Inlet water column. Sci Data. 4:170159. doi:10.1038/sdata.2017.159.
5. Hawley AK, Torres-Beltrán M, Zaikova E, Walsh DA, Mueller A, Scofield M, Hallam SJ. 2017. A compendium of multi-omic sequence information from the saanich inlet water column. Sci Data. 4:170160. doi:10.1038/sdata.2017.160.
6. Mortazavi, A., Williams, B. A., McCue, K., Schaeffer, L., and Wold, B. 2008. Mapping and quantifying mammalian transcriptomes by rna-seq. Nat Methods, 5(7):621-628.
7. Letunic I, Bork P. 2016. Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. Nucleic Acids Res 44(1):W242-W245. doi:10.1093/bioinformatics/btl529
8. Ortutay C, Ortutay Z. 2017. Introduction to R statistical environment, p 1-15. In Molecular Data Analysis, John Wiley & Sons, Hoboken, New Jersey, USA. 
9. Li M, Hong Y, Cao H, Gu JD. 2013. Community Structures and Distribution of Anaerobic Ammonium Oxidizing and niVA-Encoding Nitrite-Reducing Bacteria in Surface Sediments of the South China Sea. Microb Ecol. 66(2):281-296. doi:10.1007/s00248-012-0175-y
10. Hallam SJ, Torres-beltrán M, Hawley AK. 2017. Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Sci Data. 4:170158. doi:10.1038/sdata.2017.158.
11. Hannig M, Braker G, Dippner J, Jürgens K. 2006. Linking denitrifier community structure and prevalent biogeochemical parameters in the pelgial of the central Baltic Proper (Baltic Sea). FEMS Microbiol Ecol. 57(2):260-271. doi:10.1111/j.1574-6941.2006.00116.x
12. Canfield DE, Glazer AN, Falkowski PG. 2010. The Evolution and Future of Earth’s Nitrogen Cycle. Science. 330(6001):192-196. doi:10.1126/science.1186120
13. Lee JA, Francis CA. 2017. Spatiotemporal Characterization of San Francisco Bay Denitrifying Communities: a Comparison of nirK and nirS Diversity and Abundance. Microb Ecol. 73(2):271-284. doi:10.1007/s00248-016-0865-y
14. Kim OS, Imhoff JF, Wizel KP, Junier P. 2011. Distribution of denitrifying bacterial communities in the stratified water column and sediment–water interface in two freshwater lakes and the Baltic Sea. Aquat Ecol. 45(1):99-112. doi:10.1007/s10452-010-9335-7
15. Oakley BB, Francis CA, Roberts KH, Fuchsman, CA, Srinivasan S, Staley JT. 2007. Analysis of nitrite reductase (nirK and nirS) genes and cultivation reveal depauperate community of denitrifying bacteria in the Black Sea suboxic zone. Environ Microbiol. 9(1):118-130. doi:10.1111/j.1462-2920.2006.01121.x
16. Eilers, H., Pernthaler, J, Amann, R. 2000. Succession of pelagic marine bacteria during enrichment: a close look at cultivation-induced shifts. Appl. Environ. Microbiol. 66:4634-4649.
17. Flaerdh, K, Cohen, J, Kjelleberg, S. 1992. Ribosomes exist in large excess over the apparent demand for protein synthesis during carbon starvation in marine Vibrio sp. strain CCUB 15956. J. Bacteriol. 174:6780-6788.
18. Mills, HJ, Hunter, E, Humphrys, M, Kerkhof, L, McGuinness, L, Huettel, M, Kostka, JE. 2008. Characterization of nitrifying, denitrifying, and overall bacterial communities in permeable marine sediments of the northeastern Gulf of Mexico. Appl. Environ. Microbiol. 74:4440-4453.
19. Fuerst, JA, Sagulenko, E. 2011. Beyond the bacterium: planctomycetes challenge our concepts of microbial structure and function. Nature Reviews Microbiology. 9:403.
20. Kirkpatrick, JB, Fuchsman, CA, Yakushev, E, Staley, JT, Murray, JW. 2012. Concurrent activity of anammox and denitrifying bacteria in the Black Sea. Frontiers in Microbiology. 3:256.
21. Chen, J, Strous, M. 2013. Denitrification and aerobic respiration, hybrid electron transport chains and co-evolution. Biochimica Et Biophysica Acta (BBA)-Bioenergetics. 1827:136-144.
22. Isobe K, Ohte N. 2014. Ecological Perspectives on Microbes Involved in N-Cycling. Microbes and Environments. 29:4-16.
23. Martinez-Perez C, Mohr W, Loscher CR, Dekaezembacker J, Littmann S, Yilmaz P, Lehnen N, Fuchs BM, Lavik G, Schmitz RA, LaRoche J, Kuypers MM. 2016. The small unicellular diazotrophic symbiont, UCYN-A, is a key player in the marine nitrogen cycle. Nature Microbiology. 1:16163. doi:10.1038/nmicrobiol.2016.163.
24. Schmid M, Walsh K, Webb R, Rijpstra WIC, van de Pas-Schoonen K, Verbruggen MJ, Hill T, Moffett B, Fuerst J, Schouten S, Damsté JSS, Harris J, Shaw P, Jetten M, Strous M. 2003. Candidatus "Scalindua brodae", sp. nov., Candidatus "Scalindua wagneri", sp. nov., Two New Species of Anaerobic Ammonium Oxidizing Bacteria. Syst Appl Microbiol. 26(4):529-538. 
25. Güven D, Dapena A, Kartal B, Schmid MC, Maas B, van de Pas-Schoonen K, Sozen S, Mendez R, Op den Camp HJM, Jetten MSM, Strous M, Schmidt I. 2005. Propionate Oxidation by and Methanol Inhibition of Anaerobic Ammonium-Oxidizing Bacteria. Appl Environ Microbiol. 71(2):1066-1071. doi:10.1128/AEM.71.2.1066-1071.2005.
26. Jetten MSM, Strous M, van de Pas-Schoonen K, Kramer EHM, Muyzer G, Fuerst JA, Kuenen JG, Webb R, Logemann S. 1999. Missing lithotroph identified as new planctomycete. Nature. 400(6743):446-449. doi:10.1038/22749.