---
title: "Writing Assignment 03"
author: "Ryan Kaien Nah (28632115)"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output: html_document
---

### Module 03 Writing Assignment

**Prompt:** *"Discuss the challenges involved in defining a microbial species and how HGT complicates matters, especially in the context of the evolution and phylogenetic distribution of microbial metabolic pathways.  Can you comment on how HGT influences the maintenance of global biogeochemical cycles through time?  Finally, do you think it is necessary to have a clear definition of a microbial species?  Why or why not?"*

The species is the basic unit of biological classification, making its definition of integral importance for accurate taxonomy. The so-called species problem, or the issue with determining a completely valid species concept, has been subject to debate for centuries, with Darwin’s own *The Origin of Species* asserting that “no one [species] definition has satisfied all naturalists.” The most commonly adopted species concept, posited by Ernst Mayr (1), states that a species consists of organisms that can reproduce with one another. While this definition works reasonably for sexually reproducing organisms, it is completely non-functional for organisms which reproduce asexually, such as for most microorganisms. Even beyond the inability to use reproductive compatibility as a benchmark for microbial species membership, however, coming to a single concrete definition of what constitutes a microbial species is extremely challenging. I will begin by discussing what additional complications are inherent in the microbial species problem as compared to the species problem at the macroscopic level, such as the issue of rampant horizontal gene transfer (HGT). I will then conclude by examining whether an unambiguous microbial species definition is necessary for the discipline of microbiology.
 
Even with the relative ease of defining what constitutes a species, inconsistency of taxonomy and phylogeny is present even at the macroscopic level. For instance, many commonly-known taxonomic groups are not strictly monophyletic. For instance, Reptilia, the taxonomic class describing reptiles, encompasses all organisms of the Amniota clade except for those which are of the Mammalia or Aves class, making it paraphyletic. Numerous other taxa are paraphyletic, such as fish, monkeys, and wasps. This is not a major problem, as these organisms are clearly phenotypically distinct between species and similar within a species. This however, is a great problem at the microscopic level, as phenotypic distinction can be mediated by a small handful of genes carried by an individual plasmid or pathogenicity island. Such easily-acquired genetic differences associated with dramatic functional differences arguably should not be reason for alternate species designations. However, many microorganisms’ taxonomic assignments were made long before widespread adoption of genomic sequencing techniques. Thus, the hazy definition of what constituted a microbial species at the time has resulted in very closely genetically related species being segregated into distinct taxa entirely unnecessarily, an issue that deserves to be addressed.
 
Horizontal gene transfer more generally causes major issues for species definitions. If sizeable chunks of genetic information are transferable between genomes, bypassing vertical transmission, even using genetic similarity as a benchmark for species definitions begins to fall apart at some point. The amount of genetic information in any one given microorganism’s genome which has originated from other microorganisms is tremendous (7). 
 
The possibility of HGT essentially guarantees that if a gene or fragment of genetic material confers a fitness advantage in an environment, that it will be maintained. Thus, the maintenance of global biogeochemical cycles through time is essentially assured, making microbes the so-called guardians of metabolism. However, this possibility also means that while an individual species may not perform a given role in an environment, the overall functional output of the local biosphere as a whole will involve that role. Thus, though the original precursors of photosynthetic life have long since died off, we still feel their presence quite literally in the air around us - the genes necessary for oxygenic photosynthesis that they carried long ago now live in the plants, trees, and more importantly, ocean-dwelling *Cyanobacteria* surrounding us as humanity. While the individual has perished, the function survives. This, however, means that the associated gene/operon is likely disseminated widely throughout the community - clearly not all of these organisms can be considered to be related to one another, though they share a common genetic element!
 
Such issues become only more pronounced when examining microbial metabolic pathways in the environment. Genomic streamlining and distribution of metabolism across members of the community where each microbial community is adapted specifically to the environmental conditions with which it has existed under for perhaps millions of years (8). This means that any given bacterium likely does not exist in the exact same state anywhere else on the planet, or even within a given environment (6). As another example, take the *E. coli* bacterium, which in nature exists largely in the intestine of animal hosts, and has an incredibly diverse array of species, strains, and serotypes associated with it (2). An entire genus, *Shigella*, is composed of essentially a collection of strains of *E. coli* which arose by obtaining the pINV virulence plasmid several times, independently of one another (3). From here on, even subdivisions of species aren’t kept consistent between bacterial species - some bacteria have substrains, others have subspecies, and there are even serovars or serotypes for some. 

The issue then arises that the importance of the clarity of a microbial species definition varies greatly within individual disciplines of microbiology. For instance, the roots of microbiology are in its clinical applications, and thus many “incorrect” species definitions with archaic ancestry pre-dating genome sequencing technology possess great utility in medical microbiology. That is, when bacteria merely carrying different pathogenicity islands can cause infections of vastly different severity and pathology, clinical differentiation can become the difference between life and death. What should a physician care whether their patient’s infection is *Shigella* or *Escherichia* so long as they know what it is and how to treat it? Existing species definitions are so deeply entrenched in medical microbiology that introducing an alternate scheme for species definitions may have dire consequences. 
 
However, for an environmental microbiologist, clear definitions of what constitutes a species are absolutely essential. Organisms for which taxonomic classification does not align accurately with phylogeny is a problem of paramount importance for modern environmental microbiology, where taxonomy and phylogeny of environmental bacteria is inferred from genetic sequence similarity assuming the two are one and the same. As established before, however, the existence of paraphyletic groups like *Escherichia* and *Bacillus*, which encompasses the *Paenibacillus* and *Brevibacillus* clades (4), definitively show that is untrue. Other examples include *Azotobacter vinelandii*, which has been described as a *Pseudomonas* species in disguise due to its strong genetic similarity with presently-characterized strains of *Pseudomonas* (5). Additionally, the genetic “promiscuity” associated with environmental bacterial communities further highlights the importance of species definitions to environmental microbiology - frequent and extensive exchange of genetic material as well as redundant dispersal of metabolic function mean that any given organism adapted to its environment and associated microbial community may constitute its own strain or ecotype if not an entire species given an insufficiently broad species definition. Without clear definitions of what constitutes a microbial species, taxonomy loses its association with phylogeny, invalidating much of its purpose. 
 
In summary, the present working microbial species definition is lackluster at best, and grossly misleading at worst. It is often based largely on functional differences rather than genetic, which is a very poor way of discriminating species. Even genetic approaches fall apart when considering environmental heterogeneity in genomic content. Genetics at the microbial level is very far-removed from that at the macroscopic level, and it is long overdue for our microbial species definitions to catch up. 
 
#### References
1. **Mayr E**. 1942. Systematics and the origin of species from the viewpoint of a zoologist. Columbia University Press, New York, NY.
2. **Lukjancenko O, Wassenaar TM, Ussery DW.** 2010. Comparison of 61 sequenced *Escherichia coli* genomes. Microb Ecol **60**:708-720. 
3. **Lan R, Reeves PR.** 2002. *Escherichia coli* in disguise: molecular origins of *Shigella*. Microbes Infect **4**:1125-1132. 
4. **Xu D, Côté JC.** Phylogenetic relationships between *Bacillus* species and related genera inferred from comparison of 3′ end 16S rDNA and 5′ end 16S–23S ITS nucleotide sequences. Int J Syst Evol Microbiol **53**:695-704.
5. **Rediers H, Vanderleyden J, de Mot R.** 2004. *Azotobacter vinelandii*: a *Pseudomonas* in disguise? J Microbiol **150**:1117-1119. doi:[10.1099/mic.0.27096-0](https://dx.doi.org/10.1099/mic.0.27096-0).
6. **Thompson JR, Pacocha S, Pharino C, Klepac-Ceraj V, Hunt DE, Benoit J, Sarma-Rupavtarm R, Distel DL, Polz MF.** 2004. Genotypic diversity within a natural coastal bacterioplankton population. Science **307**:1311-1313. doi:[10.1126/science.1106028](https://dx.doi.org/10.1126/science.1106028).
7. **Welch RA, Burland V, Plunkett G 3rd, Redford P, Roesch P, Rasko D, Buckles EL, Liou SR, Boutin A, Hackett J, Stroud D, Mayhew GF, Rose DJ, Zhou S, Schwartz DC, Perna NT, Mobley HL, Donnenberg MS, Blattner FR.** 2002. Extensive mosaic structure revealed by the complete genome sequence of uropathogenic *Escherichia coli*. Proc Natl Acad Sci U S A **99**:17020-17024. [PMC139262](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC139262).
8. **Morris JJ, Lenski RE, Zinser ER.** The black queen hypothesis: evolution of dependencies through adaptive gene loss. mBio **3**:e00036-12. doi:[10.1128/mBio.00036-12](http://dx.doi.org/10.1128/mBio.00036-12).